module/VuFind/tests/unit-tests/src/VuFindTest/RecordTab/VersionsTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Versions Test Class
+ *
+ * PHP version 7
+ *
+ * Copyright (C) Villanova University 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+namespace VuFindTest\RecordTab;
+
+use Laminas\Config\Config;
+use VuFind\RecordTab\Versions;
+
+/**
+ * Versions Test Class
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Sudharma Kellampalli <skellamp@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+class VersionsTest extends \PHPUnit\Framework\TestCase
+{
+    /**
+     * Test getting Description.
+     *
+     * @return void
+     */
+    public function testGetDescription(): void
+    {
+        $count=5;
+        $som = $this->getMockPluginManager();
+        $config = $this->getMockConfig();
+        $recordDriver = $this->getMockBuilder(\VuFind\RecordDriver\SolrDefault::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+        $recordDriver->expects($this->any())->method('tryMethod')
+            ->with($this->equalTo('getOtherVersionCount'))
+            ->will($this->returnValue($count));
+        $obj= new Versions($config, $som);
+        $obj->setRecordDriver($recordDriver);
+        $translator = $this->getMockBuilder(\Laminas\I18n\Translator\TranslatorInterface::class)
+            ->getMock();
+        $translator->expects($this->any())->method('translate')
+            ->with($this->equalTo('other_versions_title'), $this->equalTo('default'))
+            ->will($this->returnValue("Count:%%count%%"));
+        $obj->setTranslator($translator);
+        $obj->getDescription();
+        $this->assertEquals("Count:5", $obj->getDescription());
```

Comment:
It would probably make sense for this to be "Count:$count" rather than "Count:5" to tie things together more explicitly.